api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/TPTListOfPatientsEligibleDataSet.java

```diff
@@ -44,6 +44,9 @@
   @Autowired private TPTEligiblePatientListCohortQueries tPTEligiblePatientListCohortQueries;
   @Autowired private HivMetadata hivMetadata;
 
+  @Autowired
```

Comment:
Can you please initialize those variables in a public constructor as opposed to having separate @Autowired?